benchmarks/src/main/scala/bloop/HotBloopBenchmark.scala

```diff
@@ -0,0 +1,109 @@
+package bloop
+
+import java.io._
+import java.nio.file._
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations.Mode.SampleTime
+import org.openjdk.jmh.annotations._
+
+import scala.tools.nsc.BenchmarkUtils
+
+import bloop.logging.Logger
+import bloop.tasks.ProjectHelpers
+
+@State(Scope.Benchmark)
+@BenchmarkMode(Array(SampleTime))
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+@Warmup(iterations = 10, time = 10, timeUnit = TimeUnit.SECONDS)
+@Measurement(iterations = 10, time = 10, timeUnit = TimeUnit.SECONDS)
+@Fork(value = 3)
+class HotBloopBenchmark {
+  @Param(value = Array())
+  var project: String = _
+
+  @Param(value = Array())
+  var projectName: String = _
+
+  @Param(value = Array(""))
+  var extraArgs: String = _
+
+  var bloopProcess: Process = _
+  var inputRedirect: ProcessBuilder.Redirect = _
+  var outputRedirect: ProcessBuilder.Redirect = _
+  var tempDir: Path = _
+  var processOutputReader: BufferedReader = _
+  var processInputReader: BufferedWriter = _
+  var output = new java.lang.StringBuilder()
+
+  @Setup(Level.Trial) def spawn(): Unit = {
+    tempDir = Files.createTempDirectory("bloop-")
+    val configDir = Files.createDirectory(tempDir.resolve(".bloop-config"))
+
+    val path = Paths.get(s"../frontend/src/test/resources/projects/$project")
```

Comment:
What happens if the project doesn't exist? Shouldn't we report it?

Reply:
It'd crash a little after that, but let's throw an exception here.